torch/_dynamo/exc.py

```diff
@@ -216,12 +216,12 @@ class ObservedAttributeError(ObservedException):
 }
 
 
-def raise_observed_exception(e, tx, vt):
+def raise_observed_exception(e, tx):
     from .variables import BuiltinVariable
 
     # CPython here raises an exception. Since there is no python code, we have to manually setup the exception
     # stack and raise the exception.
-    exception_vt = BuiltinVariable(e).call_function(vt, [], {})
+    exception_vt = BuiltinVariable(e).call_function(tx, [], {})
```

Comment:
The first argument to call_function is a InstructionTranslator object and not a VariableTracker